<launch>
    <!-- rosserial IMU  stibase_sc821 /dev/ttyUSB2-->
    <node pkg="rosserial_python" name="hc_node" type="serial_node.py" output="screen">
        <param name="port" value="/dev/ttyUSB0" />
        <param name="baud" value="57600" />
    </node>
</launch>